src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
extract random to the `private final` field:
![image](https://github.com/mate-academy/jv-lottery/assets/73041091/787e3450-9d4e-451c-875c-3effadf10e27)